src/Autocomplete/assets/package.json

```diff
@@ -13,7 +13,9 @@
                 "fetch": "eager",
                 "enabled": true,
                 "autoimport": {
-                    "tom-select/dist/css/tom-select.default.css": true
+                    "tom-select/dist/css/tom-select.default.css": true,
+                    "tom-select/dist/css/tom-select.bootstrap4.css": false,
+                    "tom-select/dist/css/tom-select.bootstrap5.css": false
```

Comment:
In another PR, I'm going to be pragmatic, though inconsistent, and remove `bootstrap4` :)